vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.cpp

```diff
@@ -32,13 +36,21 @@ DeviceInfoHolder::DeviceInfoHolder(const Mso::React::IReactContext &context) : m
 }
 
 void DeviceInfoHolder::InitDeviceInfoHolder(const Mso::React::IReactContext &context) noexcept {
```

Comment:
Is there someplace where we were previously not registering DeviceInfo when running Fabric?

Reply:
The module was being registered before, but it was crashing early on from DeviceInfoHolder being null since creation was only done under `if (xaml::TryGetCurrentApplication())`